isc_joy/mappings/blank_mapping.yaml

```diff
@@ -0,0 +1,63 @@
+# a blank mapping to serve as a template for new configurations
+# Type refers to whether a control is a Button (0) or an Axis (1)
```

Comment:
Would it be possible to use the words "button" and "axis" instead of a 0 or 1?

Reply:
yes but then i don't get to use a switch case :(

Reply:
Updated the code to use button and axis. Stripped away my beautiful switch-case setup. You monster.

Reply:
You can still use a switch case with strings can't you?

Reply:
No that's Java